rust/hyperlane-base/Cargo.toml

```diff
@@ -21,12 +22,14 @@ ed25519-dalek.workspace = true
 ethers.workspace = true
 eyre.workspace = true
 fuels.workspace = true
+hyper.workspace = true
```

Comment:
hyper shouldn't be required explicitly. You're currently using it to build the `OpenMetrics` body format but you can just take advantage of axum and remove this